assets/src/edit-story/components/panels/panel/shared/content.js

```diff
@@ -31,7 +31,7 @@ import { useContext } from 'use-context-selector';
 import panelContext from '../context';
 
 const Container = styled.div`
-  padding: ${({ padding }) => padding || '10px 16px'};
+  padding: ${({ padding }) => padding || '4px 16px 10px'};
```

Comment:
@miina, this seems like a general necessary fix? The panel content spacing is too big as it is right now, no?

Reply:
Seems so. The padding-bottom seems to be too small (should be 24):
![Screenshot 2021-03-03 at 13 04 26](https://user-images.githubusercontent.com/3294597/109796716-cb7da300-7c18-11eb-9751-f7241ffe136a.png)

![Screenshot 2021-03-03 at 13 02 48](https://user-images.githubusercontent.com/3294597/109796703-c7ea1c00-7c18-11eb-99b5-51abe52bb92c.png)

Reply:
Made some extra adjustments here - including to the panel title. This looks correct now in my opinion - have about you (ignore the border color, font family, and the % sign for now)?

| Design | Implementation |
|-|-|
| ![Screen Shot 2021-03-03 at 09 32 40](https://user-images.githubusercontent.com/637548/109820931-76836200-7c03-11eb-9a05-5d102dd841b6.png) | ![Screen Shot 2021-03-03 at 09 32 27](https://user-images.githubusercontent.com/637548/109820947-78e5bc00-7c03-11eb-9567-21782f147df9.png) |

Reply:
Looks correct, however -- maybe you can wrap the input with a `Row`? Since many other panels (due to more than one row of inputs) are using `Row` as a wrapper and that has 16px of bottom margin then this will make a difference.

If using `Row` as a wrapper, it would be 24-16 => 8px.

If not using Row, then we might need to add `isLast` or something like that to the `Row` to indicate not adding extra margin.

Or well, whatever we do inside the panel contents, would be good to have it consistent.

Thoughts?

Reply:
Makes sense to use a row everywhere, yes. I'll reintroduce it.